fdbclient/ManagementAPI.actor.cpp

```diff
@@ -484,13 +484,18 @@ ACTOR Future<ConfigurationResult::Type> changeConfig( Database cx, std::map<std:
 				tr.addReadConflictRange( singleKeyRange(m.begin()->first) );
 			}
 
-			for(auto i=m.begin(); i!=m.end(); ++i)
+			for(auto i=m.begin(); i!=m.end(); ++i) {
+				// Debug purpose
+				TraceEvent("ChangeConfigAPI").detail("Param1", i->first).detail("Param2", i->second);
 				tr.set( StringRef(i->first), StringRef(i->second) );
+			}
 
 			tr.addReadConflictRange( singleKeyRange(moveKeysLockOwnerKey) );
 			tr.set( moveKeysLockOwnerKey, versionKey );
 
 			wait( tr.commit() );
+			// Debug purpose
```

Comment:
Consider remove the debug purpose code? Probably also remove the comment'd out `printf`s.

Reply:
That's on purpose. This PR still has weird bugs to debug.